README.md

```diff
@@ -1,25 +1,54 @@
-# Pod.Cast 🎱 🐋
+# OrcaHello: A real-time AI-assisted killer whale notification system 🎱 🐋
 
-Temporary space for Orcasound ML efforts, until we refactor a bit and add to Orcasound. 
+[Orcasound](https://www.orcasound.net/) has set up a hydrophone network in the Puget Sound. Killer whales sometimes swim by these hydrophones and vocalize in the form of calls.
 
-See `./data_ml` for details on the ML side, where the documentation effort has started. 
+We trained a deep learning model to find these calls in hydrophone audio. Each overlapping 2 second sub-segment is classified as whale call/ no-whale call.
+Shown below is a 1-minute segment of hydrophone audio visualized as a spectrogram with whale calls detected by the model.
 
-## Subprojects
+![Detections](Detections.png)
```

Comment:
I recommend putting the images into a separate folder (such as img/detections.png).